src/hotspot/share/runtime/osThread.cpp

```diff
@@ -38,7 +38,7 @@ OSThread::~OSThread() {
 
 // Printing
 void OSThread::print_on(outputStream *st) const {
-  st->print("nid=%d ", thread_id());
```

Comment:
Just update the (C) year above from 2019 to 2021.
JhsdbThreadInfoTest.java has it already in the latest revision.

Reply:
Updated in both osThread.cpp and test file.